crates/turborepo-lib/src/config/mod.rs

```diff
@@ -561,22 +566,16 @@ mod test {
             vercel_artifacts_owner.into(),
         );
 
-        let override_config = ConfigurationOptions {
-            token: Some("unseen".into()),
-            team_id: Some("unseen".into()),
-            ..Default::default()
-        };
-
         let builder = TurborepoConfigBuilder {
             repo_root,
-            override_config,
+            override_config: Default::default(),
```

Comment:
@chris-olszewski my reading of the code is that after the changes in this PR there's no need for this struct property (`override_config`) at all.  I removed it in a commit, but then later dropped it because I wanted to check with you first.

Reply:
:+1: